doc/ref_model/chapters/chapter01.md

```diff
@@ -47,12 +47,12 @@ The document starts from the abstract and as it progresses it increasingly gets
 
 - **Chapter 01 - Introduction**: Overall scope of the Reference Model document including the goals and objectives of the project.  
   >	**Audience**: This chapter is written for a general technical audience with interest in this topic.
-- **Chapter 02 - VNF requirements & Analysis**: High level requirements and core principles needed to understand how the model was developed.  Addresses the thinking behind the decisions that were made.
+- **Chapter 02 - Workload requirements & Analysis**: High level requirements and core principles needed to understand how the model was developed.  Addresses the thinking behind the decisions that were made.
   >	**Audience**: This chapter is written for architects and others with an interest in how the decisions were made.
 - **Chapter 03 - Modelling**:  The high-level cloud infrastructure model itself.  
   >	**Audience**: This chapter is written for architects and others who wants to gain a quick high-level understanding of the model.
 - **Chapter 04 - Infrastructure Capabilities, Metrics, and Catalogue**:  Details about the capabilities needed to support the various types of workloads and how the capabilities are applied to the model. The details regarding T-shirt sizes and other considerations are found in this section.
-  >	**Audience**: This chapter is written for architects, developers and others who need to develop infrastructure or write VNF applications.
+  >	**Audience**: This chapter is written for architects, developers and others who need to develop infrastructure or write applications.
```

Comment:
can we change to "... deploy infrastructure and develop applications."

Reply:
> @EditKoselak the new write up states "... need to develop and deploy infrastructure or write applications ..." -- not sure about "develop infrastructure" also usually we "develop applications"
> 
> I know this is a nitpick -- sorry about that

Since CNTT's audience includes Cloud infra vendors/developers, as well, we thought keeping "develop" would be more accurate